src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1086,6 +1117,24 @@ public Visitable visit(MethodCallExpr method, Void p) {
     }
     // we will wait for the next run to solve this method call
     if (!canSolveArguments(method.getArguments())) {
+      if (hasUnsolvedReceiverLikeClassName(method)) {
+        // Make sure that we try to solve the receiver first if
```

Comment:
I would expect the `visit(ClassOrInterfaceTypeType, Void)` method to "automatically" handle (without us having to write specific lines of codes like this) the unresolved receiver if it is a class name. An example for this "automatic" handling would be the `staticmethod` test of Specimin.